packages/@react-spectrum/menu/src/MenuItem.tsx

```diff
@@ -82,11 +82,11 @@ export function MenuItem<T>(props: MenuItemProps<T>) {
             )
           }
           slots={{
-            text: {UNSAFE_className: styles['spectrum-Menu-itemLabel']},
-            end: {UNSAFE_className: styles['spectrum-Menu-end']},
+            text: {UNSAFE_className: styles['spectrum-Menu-itemLabel'], ...labelProps},
+            end: {UNSAFE_className: styles['spectrum-Menu-end'], ...descriptionProps},
             icon: {UNSAFE_className: styles['spectrum-Menu-icon']},
-            description: {UNSAFE_className: styles['spectrum-Menu-description']},
-            keyboard: {UNSAFE_className: styles['spectrum-Menu-keyboard']}
+            description: {UNSAFE_className: styles['spectrum-Menu-description'], ...descriptionProps},
```

Comment:
I don't believe spectrum allows both an `end` (e.g. key value) slot and a `description` slot in the same item, so hopefully this is safe.

Reply:
🤷‍♀ if both exist... can we do anything about it?

Reply:
we could join them in the aria-describedby? `='description-id end-id'`? VO would just concat them

Reply:
o, you did that